### ISSUE TITLE
Code problem

### ISSUE THREAD

**Issue by hongxiaDu** [kaize0409/HyperGAT#7]
Hi, thanks your work!
I have some confusion about the code.
I want to know what `self.word_context` means, and why concat with x1(`pair_h = torch.cat((q1, x1), dim=-1)`，
`q1 = self.word_context.weight[0:].view(1, -1).repeat(x1.shape[0],1).view(x1.shape[0], self.out_features)`)? 
It doesn't seem to be reflected in the formula.
![image](https://user-images.githubusercontent.com/47908560/189080843-43a95f36-24a6-44ac-ae03-c4540554d65e.png)

When AGGR(edge) aggregates features of hyperedges to nodes,` pair_h = torch.cat((q1, y1), dim=-1)` , q1 are hyperedge features, y1 are node features. So, I guess whether q1 is the hyperedge feature when nodes features aggregate to hyperedges features?
If the guess is correct? But why `self.word_context = nn.Embedding(1, self.out_features)`, instead of `self.word_context = nn.Embedding(n_hyperedge, self.out_features)`. Don't we need to distinguish features of hyperedges?
